### PR TITLE
Add workload statuses to workloads view

### LINKED ISSUE #3859: Add workload statuses to workloads view
At the moment the component is visible only in the overview.

### PR DESCRIPTION
**Pull request by medmedchiheb**
Moving component workload-status to common/components and adding it to workloads view.

/cc @maciaszczykm

Fixes #3859

### CODE REVIEW

File: src/app/frontend/resource/workloads/component.ts [hunk 1]
```diff
@@ -13,13 +13,98 @@
...
+    }
+  }
+
+  showWorkloadStatuses(): boolean {
+    return (
+        Object.values(this.resourcesRatio)
+            .reduce((sum, ratioItems) => sum + ratioItems.length, 0) !== 0);
+  }
+
+  showGraphs(): boolean {
```
Comment (maciaszczykm) [inline src/app/frontend/resource/workloads/component.ts]: Seems to be not used.

Comment (maciaszczykm) [reply src/app/frontend/resource/workloads/component.ts]: Can you remove it?

Comment (medmedchiheb) [reply src/app/frontend/resource/workloads/component.ts]: yes, i will

**Comment by maciaszczykm** [comment]
/lgtm
/approve

Let's merge anyways before the release.

**Pull request merged**